dev-envs/linux/dda.sh

```diff
@@ -2,8 +2,8 @@
 IFS=$'\n\t'
 set -euxo pipefail
 
-# Re-install dda if DDA_VERSION is set, otherwise link the dda from the conda environment
-if [[ -n "${DDA_VERSION}" ]]; then
+# Re-install dda if INSTALL_DDA is set, otherwise link the dda from the conda environment
+if [[ -n "${INSTALL_DDA:-}" ]]; then
```

Comment:
```suggestion
if [[ -n "${INSTALL_DDA}" ]]; then
```
the `{VAR:-REPL}` will get REPL if VAR is not set, so I think it's useless here.

Reply:
It's basically a way to check that it exists and it's not an empty string.